src/site/content/en/blog/learn-to-bundle/index.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Learn to bundle
+subhead: |
+    Learn to bundle your website.
+date: 2019-11-11
+hero: hero.png
+alt: A logo of a web bundle
+authors:
+  - kenjibaheux
+description: |
+    Learn to bundle your website.
+tags:
+  - post
+  - web-bundles
+---
+
+Bundling a website, and I mean a full website, in a file and making it shareable
+opens up new use cases to the web. Imagine a world where you can create your own
+content and distribute it in all sorts of ways without being restricted to the
+network. Imagine sharing a web app or piece of web content with your friends via
+Bluetooth or WiFi direct. Imagine carrying it in your own USB or even hosting it
+in your own local network. The Web Bundles API is a bleeding edge proposal that
+lets you do that.
+
+## Introducing the Web Bundles API
+
+A web bundle is a file format for encapsulating one or more HTTP resources in a
+single file, and can include one or more of HTML files, JavaScript files,
+images, and style sheets.
+
+ Web bundles, more formally known as [Bundled HTTP
+ Exchanges](https://wicg.github.io/webpackage/draft-yasskin-wpack-bundled-exchanges.html),
+ are part of the [Web Packaging](https://goto.google.com/webpackaging-one-pager)
+ proposal.
+
+<figure class="w-figure  w-figure--center">
+  <img src="webbundle.png" alt="A figure showing how web bundle bundle web resources" style="max-width: 60vw">
+  <figcaption class="w-figcaption">
+    A figure showing how web bundle bundle web resources.
+  </figcaption>
+</figure>
+
+HTTP resources in a web bundle are indexed by request URLs, and can optionally
+come with signatures that vouch for the resources. Signatures allow browsers to
+understand and verify where each resource came from, and treats each as coming
+from its true origin. This is similar to how Signed HTTP Exchanges are handled
+by browsers. ([Signed HTTP
+Exchanges](https://developers.google.com/web/updates/2018/11/signed-exchanges)
+is a feature to sign a single HTTP resource.)
+
+
+In this article I will walk through what a web bundle is and how you can use it
```

Comment:
If you're adding a second author, should this be rephrased to "we" instead of "I" ?